src/wechaty_puppet/room.py

```diff
@@ -53,3 +54,12 @@ def __init__(self):
         initialization for room payload
         """
         self.topic: str = None
+        self.owner_id: str = None
+
+
+class RoomMemberPayload:
```

Comment:
I believe there's will be better to use the `dataclass` from Python 3.7 at here.

```py
@dataclass
class RoomMemberPayload:
    room_alias: Optional[str] = None
```

Please let me know what you think, thanks.

Reply:
I agree. I will change the code later.